### ISSUE TITLE
Problems to retrieve online data (get_online_features with Redis) on v.0.28.0: it says the feature view does not exist

### ISSUE THREAD

**Issue by chrysremes** [feast-dev/feast#3457]
Contour definitions:
- A FeatureStore object named "store" is properly instatiated
- A Feature Service object named "feature_service" is properly initialized

## Expected Behavior 
By using the method "get_online_features()", we expect to retrieve data defined in feature_service.
The example uses data from open breast_cancer database, and the method looks like:
```
online_response = store.get_online_features(
    features=feature_service,
    entity_rows=[{"patient_id": 0}],
)
```
which should return (for example) : {'patient_id': [0], 'my_feature': [184.60000610351562]}

## Current Behavior
This works as expected with versions 0.27 and below. By changing to feast version 0.28, we started to see an error
```
feast.errors.FeatureViewNotFoundException: Feature view d does not exist in project feature_repo
```
Important: the same feature_service is used to retrieve data from the offline store, and this process specifically works as expected.
In other words: "get_historical_features" works as expected, while "get_online_features" not, for the same features as input.
We already tried to use names of Feature Views and Features instead of the Feature Service, but without success.

## Steps to reproduce
- Using PostgreSQL for Offline Store and Registry, hosted on Azure
- Using Azure Cache for Redis for the Online Store
- Try to reproduce the same environment and then check if the aforementioned code works on Feast v0.28

### Specifications

- Version: 0.28
- Platform: Windows NT x64 10.0.17763
- Subsystem: Python 3.9.4 x64

**Comment by fura95** [comment]
Have the same issue when I make a request `store.get_online_features()` (on Feast >= v0.27.1)
Using PostgreSQL for Offline and Online Store

**Comment by Crazynovatech** [comment]
Is the fix part of any latest release?
